packages/amplify-category-api/src/provider-utils/awscloudformation/pipeline-with-awaiter.ts

```diff
@@ -192,6 +192,43 @@ export class PipelineWithAwaiter extends Construct {
       } as Record<string, codebuild.BuildEnvironmentVariable>,
     );
 
+    const ecsDeployActionRole = new iam.Role(scope, 'EcsDeployActionRole', {
```

Comment:
Please add a TODO to remove this if/when we are able to upgrade to CDK >= 2.132.0, and add a corresponding acceptance criterion to our CDK upgrade backlog item